bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/db/JdbcBaseDAO.java

```diff
@@ -109,13 +155,15 @@ private void initSqlTypes() {
         sqlTypes.put("CALLITEM", "VARCHAR(200)");
         sqlTypes.put("COLORITEM", "VARCHAR(70)");
         sqlTypes.put("CONTACTITEM", "VARCHAR(6)");
-        sqlTypes.put("DATETIMEITEM", "DATETIME");
+        sqlTypes.put("DATETIMEITEM", "TIMESTAMP");
```

Comment:
PostgreSql and HsqlDB used "TIMESTAMP" before.
Derby, H2, MySql, Mariadb, Sqlite used "DATETIME".

I did Tests with Tables created with DATETIME and wrote TIMESTAMPs. 
Normally this causes no problems. 
However, side effects may occur with certain configured timezones. From that Time I replace the bundle, it would then save values with the correct timezone offset.